src/server/location.rs

```diff
@@ -414,4 +442,32 @@ mod test {
         assert!(loc.country() == *"US");
         Ok(())
     }
+
+    #[actix_rt::test]
+    async fn test_default_loc() -> HandlerResult<()> {
+        // From a bad setting
+        let mut settings = Settings {
+            fallback_location: "Us".to_owned(),
+            adm_endpoint_url: "http://localhost:8080".to_owned(),
+            ..Default::default()
+        };
+        assert!(settings.verify_settings().is_err());
+        settings.fallback_location = "Us, Oklahoma".to_owned();
+        assert!(settings.verify_settings().is_err());
+        settings.fallback_location = "us, Ok".to_owned();
+        assert!(settings.verify_settings().is_ok());
+        assert!(settings.fallback_location == *"USOK");
+
+        // From an empty Google LB header
+        let mut test_head = RequestHead::default();
+        let hv = "";
```

Comment:
I think we're actually gonna get back ",":

https://github.com/mozilla-services/cloudops-infra/blob/master/projects/topsites/k8s/charts/contile/templates/backendconfig.yaml#L15

(probably just used solely "{client_region_subdivision}" which _seems_ to include both but I didn't dig deep enough into what "Unicode CLDR region code" meant by country)

Reply:
I think we're good then. I altered the google header test locally to use `", "` and the code correctly returned the default.

Reply:
let me push that here and we can keep it as a proof.

Reply:
it's passing due to a nasty little bug. 

We get back "" for country and subdivision from `split()` so we have to check for those